scrapers/scraper.py

```diff
@@ -440,15 +443,21 @@ def pairData(packages, languages):
         JSON_DIR.joinpath("{}_pairData.json".format(family)), "w+", encoding="utf-8",
     )
     if not args.quiet:
```

Comment:
This would be best implemented as setting the logging level and then using `logging.info` or `logging.debug` everywhere.

Reply:
There is one problem with this, as there are warnings set for when `lexccounter` or `dixcounter` fail to count the stems for some reason, that can be turned on with `-v`. I don't think it would make sense to set this as a lower logging level that those infos, but then there would be no way of outputting that info without outputting those warnings

Reply:
Actually I could make it into debug. Would also make sense

Reply:
IIRC, you can set log levels on a per-logger basis.

Reply:
True, but come to think of it, it makes more sense as a `debug`, so it's fine either way

Reply:
Since you most likely couldn't solve the issue anyways